runtime/compiler/trj9/z/codegen/J9S390JNILinkage.hpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2018 IBM Corp. and others
```

Comment:
Copyright needs to be updated to be 2018, 2018. Similarly for the other file.

Reply:
Fixed